app/models/speaker.py

```diff
@@ -59,6 +59,7 @@ def __init__(self,
                  heard_from=None,
                  sponsorship_required=None,
                  event_id=None,
+                 user_id=None,
                  user=None):
```

Comment:
@poush 
Thanks :smile: 
Now , I am able to save the user-id .
However can't resolve this endpoint though 
** localhost:5000/v1/speakers/1/relationships/sessions**